### DISCUSSION TITLE
ELRS on R9 Stab OTA RX

### DISCUSSION THREAD

**Discussion by pHASShon** [ExpressLRS/ExpressLRS#601] [Q&A]
Hi guys! total newbe here:

I don't know if I'm the only R9 Stab OTA owner ( https://www.frsky-rc.com/r9-stab-ota/ ), but I REALY like to test the ELRS with my plane.
I haven't seen any post or comments about this RX, so I would like to ask about your thoughts about getting this updated in a different way... the right way.

I'm planning to use it with my TX16s so it colud be a total win

Tottaly appreciated (sorry for my rusty english, is not my native languaje)

Best regards
Blemysh

**Comment by greenbigfrog** [comment]
https://github.com/ExpressLRS/ExpressLRS/issues/591

**Reply by pHASShon** [reply]
Thanks, found that I'm not the only one with the RX 😆..
